plugins/google_search/help.md

```diff
@@ -29,38 +33,74 @@ This action is used to return URLs from a Google Search.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|lang|string|en|True|Language, default is en|None|
-|start|integer|1|True|First result to retrieve, default is 1|None|
-|pause|float|1.0|True|Lapse to wait between HTTP requests. A lapse too long will make the search slow, but a lapse too short may cause Google to block your IP. Your mileage may vary|None|
-|num|integer|10|True|Number of results per page, default is 10|None|
-|query|string|None|True|Query string. Must NOT be URL-encoded|None|
-|stop|integer|16|True|Number of results to retrieve to limit amount, default is 16 and the search will allows return at least 16 results if available. This option seems to have odd behavior|None|
-|only_standard|boolean|False|True|Only returns the standard results from each page. If false, it returns every possible link from each page, except for those that point back to Google itself|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|lang|string|en|True|Language, default is en|None|en|
+|num|integer|10|True|Number of results per page, default is 10|None|10|
```

Comment:
Doesnt the api support start anymore?

Reply:
Oddly start was previously listed in the help.md but the input did not exist and was not used in the plugin.